validation/types.go

```diff
@@ -1,4 +1,4 @@
-package main
+package validation
```

Comment:
Let's put this in another directory/package so when others use it they dont use `validate.Vuln`. Maybe something like `vuln` (not to be confused with the `vulns` directory which already exists)

Reply:
But isn't vuln.Vuln more confusing?

Reply:
Thinking about it from a user of this library, I think it makes more sense than `validate.Vuln`. I'd also be ok with a `types/` directory and putting it in a `types` package. Maybe let's do that

Reply:
> Thinking about it from a user of this library, I think it makes more sense than `validate.Vuln`. I'd also be ok with a `types/` directory and putting it in a `types` package. Maybe let's do that

That makes sense. I will do that.